src/component/felles/status-merkelapp/StatusMerkelapp.tsx

```diff
@@ -38,7 +38,7 @@ interface StatusProps {
 export const StatusMerkelapp = (props: StatusProps) => {
 	const { type } = props.status
 	return(
-		<Tag variant="info" size="small" className={classNames(getStyle(type), props.erDeltakerlisteVisning ? deltakerlisteStyle(type) : undefined)}>
+		<Tag variant="info" size="xsmall" className={classNames( getStyle( type ), props.erDeltakerlisteVisning ? deltakerlisteStyle( type ) : undefined )}>
```

Comment:
nit: Du har noen litt annerledes innstillinger for space ser det ut som :) Pleier ikke ha space i funksjonskall